### PR TITLE
slatelineup for firefox android home recommendations

### PR DESCRIPTION
**Pull request by jpetto**
# Goal

create a slatelineup for firefox android home recommendations. 

PRD:
* https://docs.google.com/document/d/1-P-Q15nHMGy_TQuwFh3YW07-dINXB5_rDMocwTuFl5s/edit#heading=h.jyahmgu05z6p (direct link to slates requested for this lineup)

### CODE REVIEW

**Comment by jpetto** [comment]
@mmiermans i don't think i changed anything that would cause this test to fail? do you have any quick insight here?

**Review by mmiermans — DISMISSED** [review]
Sorry for the flaky test. Fix for that is here: https://github.com/Pocket/recommendation-api/pull/338

File: app/json/slate_lineup_configs.json [hunk 1]
```diff
@@ -513,5 +513,27 @@
         ]
       }
     ]
+  },
+  {
+    "id": "a26db39b-a68a-4e01-b680-443a466f9c36",
```
Comment (jpetto) [inline app/json/slate_lineup_configs.json:518]: i just generated this in the python console - is that okay?

Comment (chelseatroy) [reply app/json/slate_lineup_configs.json:518]: Yes. At some point we discussed these IDs being a hash of the object, but then we truncated it, at which point it doesn't represent the object anymore. Random seems fine.

**Review by chelseatroy — APPROVED** [review]
Slivin'

File: app/json/slate_lineup_configs.json (hunk 1 above)
Comment (iwsmith) [reply app/json/slate_lineup_configs.json:518]: Just to share knowledge: the slate/lineup ids are static GUIDs, the only requirement is that they are unique w.r.t the namespace. The experiment_ids are truncated hashes based on the slate/lineup configuration, and are auto-generated.

**Review by kkyeboah — APPROVED** [review]
lgtm

**Pull request merged**